Spatial.gd

```diff
@@ -565,9 +565,15 @@ func _guildpoint_categories_to_godot_nodes(item: TreeItem, guildpoint_category:
 		category_name = "No Name"
 	category_item.set_text(0, category_name)
 	category_item.set_cell_mode(1, TreeItem.CELL_MODE_CHECK)
-	category_item.set_checked(1, Settings.local_category_data.get(Marshalls.raw_to_base64(category_data.guildpoint_category.get_id()), {}).get("checked", false))
+	var checked = true
+	var collapsed = true
+	if depth in [0, 1]:
+		checked = false
+	category_item.set_checked(1, Settings.local_category_data.get(Marshalls.raw_to_base64(category_data.guildpoint_category.get_id()), {}).get("checked", checked))
 	category_item.set_tooltip(1, "Show/Hide")
 	category_item.set_editable(1, true)
+	if depth == 0:
+		collapsed = false
```

Comment:
Lets move this closer to the instantiation of collapsed, or the collapsed instantiation closer to this. There is a weird nebulous zone between 569 and 575 where collapsed will be `true` even if `depth == 0`